### PR TITLE
Add new variables to support version upgrade

### PR DESCRIPTION
**Pull request by jonnyshaw89**
Running this branch also tests

https://github.com/ONSdigital/eq-terraform-dynamodb/pull/4
https://github.com/ONSdigital/eq-ecs-deploy/pull/11
https://github.com/ONSdigital/eq-survey-runner-deploy/pull/18
https://github.com/ONSdigital/eq-author-deploy/pull/11

### CODE REVIEW

File: survey-runner.tf [hunk 1]
```diff
@@ -61,10 +62,11 @@ module "eq-ecs" {
 }
 
 module "survey-runner-on-ecs" {
-  source                         = "github.com/ONSdigital/eq-survey-runner-deploy"
+  source                         = "github.com/ONSdigital/eq-survey-runner-deploy?ref=define-aws-version"
```
Comment (samiwel) [inline survey-runner.tf]: If I merge the other PRs first, this shouldn't rely on a branch?

File: survey-runner.tf [hunk 2]
```diff
@@ -88,13 +90,14 @@ module "survey-runner-on-ecs" {
 }
 
 module "survey-launcher-for-elastic-beanstalk" {
-  source                 = "github.com/ONSdigital/eq-ecs-deploy?ref=v1.1.0"
+  source                 = "github.com/ONSdigital/eq-ecs-deploy?ref=fix-version"
```
Comment (samiwel) [inline survey-runner.tf]: same as above

File: survey-runner.tf [hunk 3]
```diff
@@ -124,13 +127,14 @@ module "survey-launcher-for-elastic-beanstalk" {
 }
 
 module "survey-launcher-for-ecs" {
-  source                 = "github.com/ONSdigital/eq-ecs-deploy?ref=v1.1.0"
+  source                 = "github.com/ONSdigital/eq-ecs-deploy?ref=fix-version"
```
Comment (samiwel) [inline survey-runner.tf]: same

**Review by samiwel — CHANGES_REQUESTED** [review]

File: survey-runner.tf (hunk 1 above)
Comment (jonnyshaw89) [reply survey-runner.tf]: Correct. i left these in for easy testing

**Review by samiwel — APPROVED** [review]

**Review by dwyeradam — APPROVED** [review]

**Pull request merged**